src/operator/subgraph/mkldnn/mkldnn_fc.cc

```diff
@@ -195,6 +195,13 @@ void SgMKLDNNFCOp::Forward(const OpContext &ctx,
   }
 
   MKLDNNFCForwardFullFeature(full_param_, ctx, fwd_.get(), new_inputs, new_req, out_data);
+
+  if (mkldnn_param.quantized && !mkldnn_param.enable_float_output) {
```

Comment:
add comments on why

Reply:
I think it's straightforward here, the OutMin and OutMax are only valid when the op is quantized and not generating fp32 output.